nixnet/_session/frames.py

```diff
@@ -22,6 +22,25 @@ def __repr__(self):
     def _create_item(self, handle, index, name):
         return Frame(handle, index, name)
 
+    @property
+    def payld_len_max(self):
```

Comment:
I think this has to do with the Jenkins CI not running everytime a PR is posted or even sometimes when a PR is merged. @lilesj do you know why the CI didn't run this time?